### DISCUSSION TITLE
PlotlyChart module keeps loading and cannot display the chart properly

### DISCUSSION THREAD

**Discussion by anine09** [flet-dev/flet#701] [Q&A]
When I try to use the [example](https://github.com/flet-dev/examples/blob/main/python/controls/charts/plotly-linechart.py) on the official website, the program keeps loading and won't display the chart properly for a long time.

![image](https://user-images.githubusercontent.com/30095629/206831847-2bbd3177-7bf9-41ce-9280-d6995296c253.png)

**Comment by ndonkoHenri** [comment]
Every time you run the program?

**Reply by anine09** [reply]
yep! So what do I need to provide more information?

**Reply by anine09** [reply]
my `flet` package version is `0.3.0.dev923`
